app/screens/select_team/select_team.js

```diff
@@ -241,6 +242,21 @@ export default class SelectTeam extends PureComponent {
             );
         }
 
+        if (this.props.currentUserIsGuest) {
+            return (
+                <View style={styles.container}>
+                    <StatusBar/>
+                    <View style={styles.headingContainer}>
+                        <FormattedText
+                            id='mobile.select_team.guest_cant_join_team'
+                            defaultMessage={'You don\'t belong to any team and as a guest you can\'t join any team by yourself, you need to be invited or added by som  eone else.'}
```

Comment:
@jespino for desktop we had the user logged out with a message stating: "Your guest account has no channels assigned. Please contact an administrator. " Can we follow the same here?

Reply:
@wiersgallak i kind of prefer this approach as the reality is that the user will be able to login again even if they don’t have channels or belong to any team

Reply:
@wiersgallak Yes, actually we talk about it, and we decide to have this kind of "no-team-avaialble" approach, is the current approach in desktop too, but because we don't have an specific design yet, we are using the warning message already existing interface there.

Reply:
@jespino Please update the text to be "Your guest account has no teams or channels assigned. Please contact an administrator."

Reply:
Done! :)